docs/conf.py

```diff
@@ -252,10 +252,9 @@
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
 #texinfo_show_urls = 'footnote'
 
-intersphinx_mapping = {
-    'https://docs.python.org/': None,
-    'https://zopeschema.readthedocs.io/en/latest': None,
-}
+# intersphinx_mapping = {
+#     'python': ('https://docs.python.org/', None),
+# }
```

Comment:
```suggestion
# intersphinx_mapping = {
#     'python': ('https://docs.python.org/', None),
# }
```

Reply:
I think this intershinx reference was used in many places because the documentation mentions it and to many people it seemed like a good idea to do without checking if it makes any sense.

Reply:
I removed it hoping to get the hanging in the docs build fixed but it was not the case. But as it was marked as deprecated, I did not re-add it as it is unused and will break in Sphinx 8 if I remember correctly.